src/pages/edge/adobe-journey-optimizer-decisioning/tabs/api-reference.md

```diff
@@ -140,6 +140,53 @@ Optimize.getPropositions(scopes, new AdobeCallbackWithError<Map<DecisionScope, O
 });
 ```
 
+<Variant platform="android" api="get-propositions-withTimeout" repeat="6"/>
```

Comment:
You will need to reference this in `/adobe-journey-optimizer-decisioning/api-reference.md`